modular_skyrat/code/game/objects/items/misc.dm

```diff
@@ -2,16 +2,16 @@
 /obj/item/clothing/suit/armor/hazard
 	name = "armored hazard vest"
 	desc = "A hazard vest with plasteel and metal plates taped on it. It offers minor protection against kinetic damage, but slows you down a significant bit."
-	icon = 'modular_skyrat/icons/obj/clothing/suits.dmi'
-	mob_overlay_icon = 'modular_skyrat/icons/mob/clothing/suit.dmi'
-	anthro_mob_worn_overlay = 'modular_skyrat/icons/mob/clothing/suit_digi.dmi'
+	icon = 'modular_skyrat/icons/obj/clothing/shitarmor.dmi'
+	mob_overlay_icon = 'modular_skyrat/icons/mob/clothing/shitarmor.dmi'
+	anthro_mob_worn_overlay = 'modular_skyrat/icons/mob/clothing/shitarmor_digi.dmi'
```

Comment:
Can you please not make extra files and name them like this.

Reply:
ok